app/components/works/detail_state_display_component.rb

```diff
@@ -11,7 +11,7 @@ def initialize(work_version:)
 
     def call
       value = I18n.t(work_version.state, scope: 'work.state')
-      return value unless work_version.wait_state?
+      return value.html_safe unless work_version.wait_state? # rubocop:disable Rails/OutputSafety
```

Comment:
do you need both `.html_safe` and the rubocop disable?

Reply:
Yes, the rubocop disable is saying "allow the use of html_safe".